### PR TITLE
feat: add onSelectOption handler

### PR DESCRIPTION
**Pull request by neolitec**
SCFRCA-1045

This PR adds a global handler to easily identify which menu item has been selected.

### CODE REVIEW

File: dist/ovh-angular-actions-menu.js [hunk 1]
```diff
@@ -280,22 +285,29 @@ angular.module("ovh-angular-actions-menu").controller("ActionsMenuCtrl", ["actio
  *      };
  *
  *      $scope.actionsOptions = [{
+ *          id: "firstButton",
  *          title: "My Beautiful title",
  *          icon: "filled-check",
  *          href: "http://www.google.be"
```
Comment (FredericEspiau) [inline dist/ovh-angular-actions-menu.js:291]: Obvious Belgian propaganda

**Review by whispyy — APPROVED** [review]

**Review by Netnix — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -100,14 +101,19 @@ This is the main directive of the module. It's creating a popover with desired a
          icon: "filled-error",
          state: "my-manager.state1"
      }];
+
+     $scope.onSelectOption = function (id) {
+        console.log("Option has been selected!", id);
```
Comment (JDownloader) [inline README.md:106]: `console.log` are not very end-user friendly, maybe adding an error message to scope could be better
```
# in js
$scope.errorMessage =`Option has been selected! ${id}`;
# in html
<div class="red" data-ng-bind="errorMessage"></dev>
```

File: dist/ovh-angular-actions-menu.js [hunk 3]
```diff
@@ -280,22 +285,29 @@ angular.module("ovh-angular-actions-menu").controller("ActionsMenuCtrl", ["actio
...
  *      }, {
+ *          id: "secondButton",
  *          title: "My Other title",
  *          icon: "filled-error",
  *          state: "my-manager.state1"
  *      }];
+ *
+ *      $scope.onSelectOption = function (id) {
+ *          console.log("Option has been selected!", id);
```
Comment (JDownloader) [inline dist/ovh-angular-actions-menu.js:300]: same `console.log`

File: docs/partials/docs/actionsMenu.directive.actionsMenu.html [hunk 4]
```diff
@@ -27,20 +27,27 @@ <h2 id="example">Example</h2>
...
         href: "http://www.google.be"
     }, {
+        id: "secondButton",
         title: "My Other title",
         icon: "filled-error",
         state: "my-manager.state1"
     }];
+
+    $scope.onSelectOption = function (id) {
+        console.log("Option has been selected!", id);
```
Comment (JDownloader) [inline docs/partials/docs/actionsMenu.directive.actionsMenu.html:42]: same `console.log`

File: dist/ovh-angular-actions-menu.js (hunk 1 above)
Comment (JDownloader) [reply dist/ovh-angular-actions-menu.js:291]: I also love google Belgium, but maybe we should not chose this as an example

**Review by JDownloader — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (neolitec) [reply README.md:106]: This is only a documentation. As a developer you understand that you can update the $scope if you want, don't you?
This is the same for the other comments.

**Pull request merged**